Common/Cloud-Account/aws-cfn-cloud-account-connector/README.md

```diff
@@ -1,6 +1,6 @@
 # Add AWS Account to Cloud One
```

Comment:
I think it is important to note here that you need to ensure the AWS Account before hand can handle the amount of S3 buckets that will be created by Sentry. I have seen this cause many failed deployments and rollbacks are even more of a challenge to navigate with Sentry.
Or IMHO, just remove sentry part

Reply:
Its not documented. The stack creates 2 S3 buckets per AWS Region that we support. There are 31 total AWS regions.